language/compiler/ir-to-bytecode/src/context.rs

```diff
@@ -328,6 +348,48 @@ impl<'a> Context<'a> {
         ))
     }
 
+    /// Get the field handle index for the alias, adds it if missing.
+    pub fn field_handle_index(&mut self, handle: FieldHandle) -> Result<FieldHandleIndex> {
```

Comment:
Same issue as `struct_instantiation_index`. Perhaps construct the `FieldHandle` inside?

Reply:
we'll do on all accounts and good call